tests/test_lua13_bdd_eval.lua

```diff
@@ -0,0 +1,61 @@
+local cloe = require("cloe")
+
+do
+    local project = require("project")
+    project.configure_all({
+        with_server = false,
+    })
+    project.init_report(require("report_config"), { foo = "bar" })
+    project.set_realtime_factor(-1)
+end
+
+-- Example schedule_test with testing using the Lust library:
+--
+--     https://github.com/bjornbytes/lust
+--
+-- Inside the run function we need to use z:describe to start
+-- using the Lust library, and we need to import `lust.it` and
+-- `lust.expect` from it.
+cloe.schedule_test {
+    id = "d7f31aaa-ccab-421b-a9ae-06aa3835018b",
+    on = "start",
+    info = { hello = "test", rqm = "big number" },
+    desc = "this is a long text for test",
+    terminate = false,
+    run = function(z, sync)
```

Comment:
What is "z"? A more descriptive name would be helpful? For example "cloe_testfixture"....

Reply:
Variable `z` is used so often it's kept short. Strongly influenced by how testing works in Go: https://pkg.go.dev/testing.

If you have the Lua extension enabled and the Language Server active, it will show you the type of `z`, incidentally.

Reply:
Having a name like `cloe_testfixture` that needs to be repeated for almost every line is something that I think leads to less readability in the end.